crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,2491 @@
+# This file was generated from /Users/timothydballard/temp/zh-hant.xlsx
+msgid ""
+msgstr ""
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-04-08 16:55+0000\n"
+"PO-Revision-Date: 2021-03-31 17:42-0500\n"
+"Language: Chinese Traditional\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: xls-to-po 1.0\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
```

Comment:
Good opportunity to make these consistent across `.po `files now and set the pattern going forward for other languages.

Reply:
Made the fields consistent between `zh-hant` and `es`, in part by removing the fields we haven't historically made use of. 

The `Report-Msgid-Bugs-To` field is recreated each time `makemessages` is run, so perhaps we should populate that with an actual email address to field these reports, however unlikely they are